extensions/amp-fit-text/1.0/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@ampproject/amp-fit-text",
+  "version": "VERSION",
+  "description": "AMP Fit Text Component",
+  "author": "The AMP HTML Authors",
+  "license": "Apache-2.0",
+  "main": "dist/component.js",
+  "module": "dist/component.mjs",
+  "exports": {
+    ".": {
```

Comment:
Same qq:
1. How would CE export look here?
2. How would React export look here? Do we need a separate export for React at all?

Reply:
> How would CE export look here?

```
"custom-element": {
  "import": "dist/custom-element.mjs",
  "require": "dist/custom-element.js",
}
```

Ditto for React.

> Do we need a separate export for React at all?

Depends on how the user is doing their bundling. Some bundlers allow remapping imports to other packages (`preact` -> `react` for them), but some would need to preprocess the npm package. I think providing a React package is good to do, because the `preact` package doesn't cleanly split into `react` and `react-dom`.

Reply:
> I think providing a React package is good to do, because the preact package doesn't cleanly split into react and react-dom

Could you clarify this? By "React package" do you mean an NPM package separate from Preact? Or are they somehow coexist inside the same NPM package?

Reply:
They coexist in this same `@ampproject/amp-fit-text` package as different files. Eg, `component.js` which is Preact, and `component-react.js` which has its imports already remapped to `react`.

Reply:
Got it!